nautobot_data_validation_engine/templates/nautobot_data_validation_engine/minmaxvalidationrule.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```

Comment:
`object_detail.html` was only added in Nautobot 1.2, so making this change will also need to raise the minimum required Nautobot version for this plugin correspondingly (`min_version` in `nautobot_data_validation_engine/__init__.py` as well as documentation).